core/events_inc.php

```diff
@@ -46,6 +46,7 @@
 	'EVENT_LAYOUT_CONTENT_END' => EVENT_TYPE_OUTPUT,
 	'EVENT_LAYOUT_PAGE_FOOTER' => EVENT_TYPE_OUTPUT,
 	'EVENT_LAYOUT_BODY_END' => EVENT_TYPE_OUTPUT,
+	'EVENT_LAYOUT_NAVBAR' => EVENT_TYPE_OUTPUT,
```

Comment:
This event name gives the impression that you can customize any part of the navbar, but the actual implementation seems to focus on the logo part - Is EVENT_LAYOUT_NAVBAR_LOGO a more appropriate name?
The idea here to allow developers to replace the default logo segment with any other custom menu.